### DISCUSSION TITLE
Include --clean, --no-cache, or --force as CLI options

### DISCUSSION THREAD

**Discussion by chista0x** [Cyfrin/aderyn#525] [Ideas]
I believe these options would be extremely beneficial for developers utilizing the tool.

**Comment by alexroan** [comment]
Hey @chista0x! Thanks for opening this discussion.

Is this to rebuild foundry output files before running the tool again?

**Reply by chista0x** [reply]
Yes. I compiled and built after changing the code. But the report still refers to the code before the change. The report is updated only when I delete the previous build and build again.

1. `forge build`
2. `aderyn`
3. generated report
4. fix bug and change code

scenarios 1: the report is not updated
1. `aderyn`

scenarios 2: the report is not updated
1. `forge build`
2. `aderyn`

scenarios 3: the report is updated
1. `forge clean` <- need this step
2. `forge build`
3. `aderyn`

**Reply by TilakMaddy** [reply]
@chista0x The latest version of `aderyn` should solve that problem because it no longer requires `forge` to create those artifacts. Therefore it will always run on the latest code. Can you give that a try? Let us know : )

**Comment by chista0x** [comment]
Yes, my previous version was `0.1.2`, which I upgraded to `0.1.3`, the problem was completely solved. Thank you for your efforts and guidance.  @TilakMaddy @alexroan
